### PR TITLE
Added enemy tracking to compass and visual tweaks

### PR DESCRIPTION
**Pull request by ikeia**
- Enemies appear as red dots on the compass now
- Added unique icons to upgrades(Placeholders for now)
- Adjusted upgrade panel

### CODE REVIEW

**Review by zhuandy42 — APPROVED** [review]
looks good! we'll look into merging this into master after playtesting tonight, so hold off on merging for now

**Comment by zhuandy42** [comment]
ok to merge whenever

**Review by MaXeraph — APPROVED** [review]

**Pull request merged**
